src/azul/service/manifest_service.py

```diff
@@ -776,6 +809,409 @@ def manifest_config(self) -> ManifestConfig:
 Bundles = MutableMapping[FQID, Bundle]
 
 
+@dataclass(frozen=True)
+class Relation:
+    # FIXME: Currently `document_id` is not sufficient to uniquely identify a
+    # PFB Entity
+    dst_id: str
+    # A more appropriate variable name would be dst_type, but we stick with name
+    # to conform to PFB spec
+    dst_name: str
+
+    @classmethod
+    def to_entity(cls, entity: 'PFBEntity'):
+        return cls(dst_id=entity.id, dst_name=entity.name)
+
+
+pfb_namespace = uuid.UUID('3372fc62-76d2-4ada-a68a-f08d4f558da5')
+
+
+@dataclass(unsafe_hash=True)
+class PFBEntity:
+    # A more appropriate variable name would be type, but we stick with name to
+    # conform to PFB spec
+    name: str
+    id: str = field(init=False)
+    object: MutableJSON = field(compare=False)
```

Comment:
Avoid `object`. Too generic and without precedence in Azul. What IS this?

Reply:
the name `object` comes from the PFB schema. The fields `name`, `id`, and `object` all do.